Libraries/Utilities/Platform.android.js

```diff
@@ -19,7 +19,10 @@ export type PlatformSelectSpec<A, N, D> = {
 
 const Platform = {
   __constants: null,
-  OS: 'android',
+  // $FlowFixMe[unsafe-getters-setters]
+  get OS(): string {
+    return 'android';
+  },
   // $FlowFixMe[unsafe-getters-setters]
   get Version(): number {
     return this.constants.Version;
```

Comment:
Just a suggestion:
Can we also try to change Version to osVersion as is the case in Platform.ios.js? It just keeps parity between two similarly used files